fc/environment.py

```diff
@@ -2,7 +2,7 @@
 import numpy as np
```

Comment:
I'm not sure about the rationale for moving this file (unlike moving protocol.py which I agree with)?

Similarly the other utility modules below.

Reply:
An environment seems like a core concept in FC to me, no? I'd think a utility was something like a CSV parser or some other stand-alone feature that's not very FC-specific but needed anyway

Reply:
On that basis, I think everything in `utility` should be moved into the top level folder!

Reply:
ok!